src/sagemaker/dataset_definition/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Placeholder docstring"""
```

Comment:
can you fill these in please?

Reply:
I'm not sure what need to be filled here, I looked at other __init__ classes, they all have """Placeholder docstring"""

Reply:
Uh found it, changed.